### PR TITLE
Add version (1.0) to API end-points, as usual in wazo-platform

### PR DESCRIPTION
**Pull request by tranchitella**
Depends-On: https://github.com/wazo-platform/wazo-c4/pull/4

### CODE REVIEW

**Review by alexsosic — DISMISSED** [review]

File: docker-compose.wazo-router-confd.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  wazo-router-confd:
+    image: wazopbx/wazo-router-confd:latest
+    ports:
+    - 8000:8000
```
Comment (sboily) [inline docker-compose.wazo-router-confd.yaml]: Please can you change the port to 8600 instead 8000. We don't want to use the default port from the debug server.  https://wazo-platform.org/contribute/new_service

**Review by sboily — DISMISSED** [review]

**Review by pc-m — COMMENTED** [review]
Is it possible to add the prefix `1.0` on the TestClient creation to avoid having to specify the version on each calls in tests?

Looks good to me, nothing surprising.

**Review by alexsosic — DISMISSED** [review]
This version is working with C4 integration tests.

File: docker-compose.wazo-router-confd.yaml (hunk 1 above)
Comment (mwolff44) [reply docker-compose.wazo-router-confd.yaml]: You can use port number 9600

Comment (tranchitella) [reply docker-compose.wazo-router-confd.yaml]: @sboily @mwolff44 8600 or 9600? how can I know which port is the next available one in the wazo-platform?

Comment (sboily) [reply docker-compose.wazo-router-confd.yaml]: sorry my mistake, it's 9600.

Comment (sboily) [reply docker-compose.wazo-router-confd.yaml]: We discussed to start at 9600 for c4.

**Comment by tranchitella** [comment]
> Is it possible to add the prefix `1.0` on the TestClient creation to avoid having to specify the version on each calls in tests?

Maybe, but to be honest, I prefer to have the full URL in the tests instead of decoupling the version number from the URL, which is going to be tested. It makes things easier to understand at first sight.

**Review by alexsosic — DISMISSED** [review]

**Comment by tranchitella** [comment]
recheck

**Review by mwolff44 — APPROVED** [review]

**Comment by tranchitella** [comment]
regate

**Pull request merged**